packages/models/src/eservice/protobufConverterFromV2.ts

```diff
@@ -148,6 +148,7 @@ export const fromDescriptorV2 = (input: EServiceDescriptorV2): Descriptor => ({
     input.templateVersionRef != null
       ? fromEServiceTemplateVersionRefV2(input.templateVersionRef)
       : undefined,
+  audience: input.audience.map((aud) => aud.replaceAll("\u0000", "")),
```

Comment:
```suggestion
// Remove null characters (\0) from PostgreSQL strings to prevent data processing issues
  audience: input.audience.map((aud) => aud.replace(/\0/g, "")),
```
I suggest to use a regex which should be quicker than a replaceAll and also to add a comment for explaining why we need it

Reply:
This is tracked in a different task: [PIN-6761](https://pagopa.atlassian.net/browse/PIN-6761)